test/integration/framework/src/siftool/run_env.py

```diff
@@ -568,9 +568,7 @@ def run(self):
         validator_count = 1
         network_definition_file = os.path.join(networks_dir, "network-definition.yml")
         seed_ip_address = "192.168.1.2"
-        mint_amount = [[999999 * 10**21, "rowan"],
-                       [137 * 10**16, "ibc/FEEDFACEFEEDFACEFEEDFACEFEEDFACEFEEDFACEFEEDFACEFEEDFACEFEEDFACE"],
-                       [10**28, "sifBridge99990x0000000000000000000000000000000000000000"]]
+        mint_amount = [[999999 * 10**21, "rowan"], [137 * 10**16, "ibc/FEEDFACEFEEDFACEFEEDFACEFEEDFACEFEEDFACEFEEDFACEFEEDFACEFEEDFACE"]]
```

Comment:
please don't remove the sifBridge99990x0000000000000000000000000000000000000000, it is used to pay cross chain fee from sifnode to ethereum.

Reply:
I removed it from class `IntegrationTestsEnvironment` which is used for exclusively for Peggy1. As far as I understand this denom is Peggy2-specific.